src/components/user/User.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const User = ({ name, followers, following, URL, avatar_url }) => (
+  <figure>
+    <img src={avatar_url} alt={name} />
+    <figcaption>
+      <p>{name}</p>
+      <p>{followers}</p>
+      <p>{following} </p>
+      <p>{URL}</p>
+    </figcaption>
+  </figure>
+);
+
+User.prototype = {
+  avatar_url: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  followers: PropTypes.string.isRequired,
```

Comment:
looks like followers and following are being passed through as numbers, not strings which is why you're getting those errors.

Reply:
weirdly enough when I was passing a string it was erroring out because a number was showing up.

Reply:
I am going to rework it based on Ryans elucidating advice out instructions today.